src/components/Team.vue

```diff
@@ -22,46 +22,58 @@ export default {
     return {
       members: [
         {
-          id: 1,
           name: 'Mauricio Pautasso',
           position: 'Co-Founder & CEO',
           photo: require('../assets/img/Mauricio-Pautasso.svg'),
           linkedin: 'https://www.linkedin.com/in/mauriciopautasso'
         },
         {
-          id: 2,
           name: 'Abel Osorio',
           position: 'Co-Founder & CTO',
           photo: require('../assets/img/Abel-Osorio.svg'),
           linkedin: 'https://www.linkedin.com/in/abelosorio89'
         },
         {
-          id: 3,
           name: 'Serguey Shevtsov',
           position: 'Full Stack Developer',
           photo: require('../assets/img/Sergio-Shevtsov.svg'),
           linkedin: 'https://www.linkedin.com/in/serguey-shevtsov'
         },
         {
-          id: 4,
           name: 'Armando Graterol',
           position: 'Frontend Developer',
           photo: require('../assets/img/Armando-Graterol.svg'),
           linkedin: 'https://www.linkedin.com/in/armandograterol/'
         },
         {
-          id: 5,
           name: 'Agostina Giordano',
           position: 'UX UI Designer',
```

Comment:
Let's change this to `UX/UI Designer & Project Manager`.

Reply:
Done!